aioftp/client.py

```diff
@@ -562,6 +737,9 @@ class AsyncClientLister(AsyncListerMixin):
                 return cls
 
             async def __anext__(cls):
+                return await cls._anext()
+
+            async def _anext_MSLD_LIST(cls):
```

Comment:
I mean, that you don't need to modify `__anext__` at all, since `__anext__` calls `cls._anext`, which is set by `_new_stream` only with one value: `cls._anext_MSLD_LIST`. So you can just change `_anext_MSLD_LIST` to `__anext__`.